.github/workflows/iroha2-dev.yml

```diff
@@ -19,10 +17,22 @@ jobs:
       image: 7272721/i2-ci:nightly
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - uses: tj-actions/changed-files@v29.0.2
```

Comment:
```
Run tj-actions/changed-files@v29.0.2
changed-files-diff-sha
  Verifying git version...
  Valid git version found: (2.34.1)
  Error: Process completed with exit code 128.
```

Reply:
Could you provide a link to the error log?

Reply:
I tried to test [on my fork](https://github.com/BAStos525/soramitsu-iroha/runs/8130280446?check_suite_focus=true)

Reply:
Do you have any alternative ideas to filter out the base image publishing @BAStos525  ?
I've on my fork tried `workflow_call` and `workflow_run`, both of them did not work

Reply:
I'd leave as is and merge, because this can be a problem specific to forks, as experiments on my fork report that both [`tj-actions/changed-files`](https://github.com/s8sato/iroha/actions/runs/3002076947) and [`dorny/paths-filter`](https://github.com/s8sato/iroha/actions/runs/3002856835) failed with exit code 128, which seems to be authentication problem

Reply:
Interesting thing here that [iroha2 longevity load-rs repository](https://github.com/soramitsu/iroha2-longevity-load-rs) is also [fork](https://github.com/e-ivkov/iroha2-longevity-load-rs). But `dorny/paths-filter` Actions works as expected.

Reply:
That would be the difference whether secrets are set properly or not

Reply:
Assume that `tj-actions/changed-files` will work inside `hyperledger` repository, does it mean that we will get the workflows run fails of forks wherever this particular Action is specified?
If third-part Actions  doesn't work for us, we could try something with a [classical bash commands](https://www.meziantou.net/executing-github-actions-jobs-or-steps-only-when-specific-files-change.htm).

Reply:
What do you think about actually merging and observing the results, because:
- This is in accordance with the usage
- The exit code 128 implies an authentication error

Reply:
Well, let's get a chance for it and see.